lib/matplotlib/axes/_axes.py

```diff
@@ -6341,10 +6330,9 @@ def pcolorfast(self, *args, alpha=None, norm=None, cmap=None, vmin=None,
             self.add_image(im)
             ret = im
 
-        if vmin is not None or vmax is not None:
-            ret.set_clim(vmin, vmax)
```

Comment:
This was actually a bug. it should have been inside the `np.dim(C) == 2` because RGB(A) data also do not need a `set_clim`.